cypress/utils/apps/query_enhancements/sidebar.js

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * Sends a new query via the query multiline editor.
+ * @param {string} query Query string
+ * @see https://docs.cypress.io/api/commands/type#Arguments
+ */
+export const sendQueryOnMultilineEditor = (query) => {
+  // remove syntax helper
+  cy.getElementByTestId('headerGlobalNav').click();
+  // Clear default text on the editor by an alternative method, since
+  // cy.clear() won't work for some reason
+  cy.get('.view-line')
+    .invoke('text')
+    .then(($content) => {
+      const contentLen = $content.length + 1;
+      cy.get('.view-line').type('a'); // make sure we're at the end of the string
+      cy.get('.view-line').type('{backspace}'.repeat(contentLen));
+    });
+  // Type query
+  cy.get('.view-line').type(query);
+  // Send query
+  cy.getElementByTestId('querySubmitButton').click();
+};
```

Comment:
is this funciton needed? can we remove it and use this instead?

https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/cypress/utils/apps/query_enhancements/commands.js#L6

Reply:
removed